src/payment/strategies/braintree/braintree.ts

```diff
@@ -368,6 +371,10 @@ export interface PaypalClientInstance {
     createPayment(data: BraintreePaypalRequest): Promise<string>;
 }
 
-export interface Config {
-    currency?: string ;
+export interface BraintreeComponents {
+    components?: string;
```

Comment:
Do we use this anywhere?

Reply:
Yes. Here `instance.loadPayPalSDK({
                        currency: config.currency,
                        components: PAYPAL_COMPONENTS.toString(),
                    },`

Reply:
`export interface Config extends BraintreeComponents {
    currency?: string;
}
`

Reply:
Yes, thanks missed that one.